src/Hedgehog/Gen.fs

```diff
@@ -551,3 +547,13 @@ module GenBuilder =
 module GenOperators =
     let (<!>) = Gen.map
     let (<*>) = Gen.apply
+
+open System
+
+/// In C# friendly manner, a generator for values and shrink trees of type 'a
+type Gen =
+    static member Bool = Gen.bool
+    static member Int32(range : Range<int>) : Gen<int> = Gen.int range
+    static member Constant(x : 'a) : Gen<'a> = Gen.constant x
+    static member Map(f : Func<'a, 'b>) (g : Gen<'a>) : Gen<'b> =
```

Comment:
Don't we have a [`Select`](https://github.com/hedgehogqa/fsharp-hedgehog/blob/ed75041fe59082153a0a659128c41cfabee2db10/src/Hedgehog/LinqSupport.fs#L19) already? C# users are more familiar with that instead of `Map`.

Reply:
I moved it from the previous code and it is better to remove it.